### ISSUE TITLE
ssconf_hvparams_* not distributed with ssconf

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#1129]
Originally reported of Google Code with ID 1073.

```
Simple way to reproduce
- remove /var/lib/ganeti/ssconf_hvparams_fake on some node
- gnt-cluster add-tags foo

...and nothing happens (gnt-cluster verify still complains
about that file missing). However, if you look into the
node daemon log (assuming debug mode), you'll see that
wconfd did a full redistribution of the ssconf files---and
the hypervisor parameters are not part of it.

```
Originally added on 2015-04-21 16:01:16 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Fixed in

commit 1f8b080ae9fe4e8b2095b07003dc990e8a69894b
Author: Petr Pudlak <pudlak@google.com>
Date:   Wed Apr 22 19:09:47 2015 +0200

    Include hypervisor parameters in SSConf
    
    This was omitted after the refactoring of SSConf to Haskell, now being
    added. Fixes #1073.
    
    Signed-off-by: Petr Pudlak <pudlak@google.com>
    Reviewed-by: Klaus Aehlig <aehlig@google.com>

commit 7c6deb9fcf93a3957032700ba67d4c2b761a88fc
Author: Petr Pudlak <pudlak@google.com>
Date:   Wed Apr 22 17:48:27 2015 +0200

    Add SSConf keys for hypervisor parameters
    
    .. and a function for converting hypervisor types to the keys.
    
    Signed-off-by: Petr Pudlak <pudlak@google.com>
    Reviewed-by: Klaus Aehlig <aehlig@google.com>

commit 812366793041fbea3976a2d6bc59185203b3f335
Author: Petr Pudlak <pudlak@google.com>
Date:   Wed Apr 22 13:10:29 2015 +0200

    Use Hypervisor as the key in ClusterHvParams
    
    .. instead of String. This imposes a type-level restriction that the
    keys of the map are just hypervisor names.
    
    Note about 'Arbitrary GroupDiskParams': Since GroupDiskParams and
    ClusterHvParams were both synonyms for 'Container (Container JSValue)',
    the Arbitrary instance worked for both. After fixing the type of
    ClousterHvParams, its instance become different from GroupDiskParams,
    hence the latter needs the addition.
    
    Signed-off-by: Petr Pudlak <pudlak@google.com>
    Reviewed-by: Klaus Aehlig <aehlig@google.com>
```
Originally added on 2015-04-28 12:59:57 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2015-04-29 10:54:44 +0000 UTC.

Changed State: Released
